d2l-table-style.html

```diff
@@ -0,0 +1,309 @@
+<link rel="import" href="../polymer/polymer.html">
+<link rel="import" href="../d2l-colors/d2l-colors.html">
+
+<!--
+This module styles table elements under <content> tags.
+
+	<style include="d2l-table-style"></style>
+	<table class="d2l-table">
+		<template>
+			<content></content>
+		</template>
+	</table>
+	<div class="d2l-table"><content></content></div>
+
+TODO: When ShadowDOM v1 is available, use <slot> and ::slotted
+-->
+<dom-module id="d2l-table-style">
+	<template>
+		<style include="d2l-colors">
+			.d2l-table {
```

Comment:
OK, I'd like to maybe explore some other options if we can, because I'm pretty sure it's illegal (in the laws of [HTML table land](https://www.w3.org/TR/html5/tabular-data.html)) to have a `<div>` as a child of `<table>`, even if the CSS `display` is set correctly. There certainly might be some accessibility repercussions/bugs, not to mention anyone through the HTML through a validator.